// @expo-google-fonts/plus-jakarta-sans/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const PlusJakartaSans_200ExtraLight = require('./200ExtraLight/PlusJakartaSans_200ExtraLight.ttf');
export const PlusJakartaSans_300Light = require('./300Light/PlusJakartaSans_300Light.ttf');
export const PlusJakartaSans_400Regular = require('./400Regular/PlusJakartaSans_400Regular.ttf');
export const PlusJakartaSans_500Medium = require('./500Medium/PlusJakartaSans_500Medium.ttf');
export const PlusJakartaSans_600SemiBold = require('./600SemiBold/PlusJakartaSans_600SemiBold.ttf');
export const PlusJakartaSans_700Bold = require('./700Bold/PlusJakartaSans_700Bold.ttf');
export const PlusJakartaSans_800ExtraBold = require('./800ExtraBold/PlusJakartaSans_800ExtraBold.ttf');
export const PlusJakartaSans_200ExtraLight_Italic = require('./200ExtraLight_Italic/PlusJakartaSans_200ExtraLight_Italic.ttf');
export const PlusJakartaSans_300Light_Italic = require('./300Light_Italic/PlusJakartaSans_300Light_Italic.ttf');
export const PlusJakartaSans_400Regular_Italic = require('./400Regular_Italic/PlusJakartaSans_400Regular_Italic.ttf');
export const PlusJakartaSans_500Medium_Italic = require('./500Medium_Italic/PlusJakartaSans_500Medium_Italic.ttf');
export const PlusJakartaSans_600SemiBold_Italic = require('./600SemiBold_Italic/PlusJakartaSans_600SemiBold_Italic.ttf');
export const PlusJakartaSans_700Bold_Italic = require('./700Bold_Italic/PlusJakartaSans_700Bold_Italic.ttf');
export const PlusJakartaSans_800ExtraBold_Italic = require('./800ExtraBold_Italic/PlusJakartaSans_800ExtraBold_Italic.ttf');
