// @expo-google-fonts/work-sans/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const WorkSans_100Thin = require('./100Thin/WorkSans_100Thin.ttf');
export const WorkSans_200ExtraLight = require('./200ExtraLight/WorkSans_200ExtraLight.ttf');
export const WorkSans_300Light = require('./300Light/WorkSans_300Light.ttf');
export const WorkSans_400Regular = require('./400Regular/WorkSans_400Regular.ttf');
export const WorkSans_500Medium = require('./500Medium/WorkSans_500Medium.ttf');
export const WorkSans_600SemiBold = require('./600SemiBold/WorkSans_600SemiBold.ttf');
export const WorkSans_700Bold = require('./700Bold/WorkSans_700Bold.ttf');
export const WorkSans_800ExtraBold = require('./800ExtraBold/WorkSans_800ExtraBold.ttf');
export const WorkSans_900Black = require('./900Black/WorkSans_900Black.ttf');
export const WorkSans_100Thin_Italic = require('./100Thin_Italic/WorkSans_100Thin_Italic.ttf');
export const WorkSans_200ExtraLight_Italic = require('./200ExtraLight_Italic/WorkSans_200ExtraLight_Italic.ttf');
export const WorkSans_300Light_Italic = require('./300Light_Italic/WorkSans_300Light_Italic.ttf');
export const WorkSans_400Regular_Italic = require('./400Regular_Italic/WorkSans_400Regular_Italic.ttf');
export const WorkSans_500Medium_Italic = require('./500Medium_Italic/WorkSans_500Medium_Italic.ttf');
export const WorkSans_600SemiBold_Italic = require('./600SemiBold_Italic/WorkSans_600SemiBold_Italic.ttf');
export const WorkSans_700Bold_Italic = require('./700Bold_Italic/WorkSans_700Bold_Italic.ttf');
export const WorkSans_800ExtraBold_Italic = require('./800ExtraBold_Italic/WorkSans_800ExtraBold_Italic.ttf');
export const WorkSans_900Black_Italic = require('./900Black_Italic/WorkSans_900Black_Italic.ttf');
